apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/AccountController.java

```diff
@@ -664,4 +672,34 @@ public Response enable2FA(
         return response.bind(dto).build();
     }
 
+    @Path("/count")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+        summary = "Returns statistics Account information",
+        description = "Returns statistics information about specified count of account",
+        tags = {"accounts"},
+        responses = {
+            @ApiResponse(responseCode = "200", description = "Successful execution",
+                content = @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = AccountsCountDto.class)))
+        }
+    )
+    @PermitAll
+    public Response counts(
+        @Parameter(name = "numberOfAccounts", description = "number Of returned Accounts, optional, minimal value = 50, maximum = 500", allowEmptyValue = true)
+        @QueryParam("numberOfAccounts") String numberOfAccountsStr
```

Comment:
@Range + @DefaultValue better to use, is not it?

Reply:
No, it works worse and not backward compatible.